src/test/test-namespace.c

```diff
@@ -100,42 +101,56 @@ static void test_shareable_ns(unsigned long nsflag) {
 
         if (pid1 == 0) {
                 r = setup_shareable_ns(s, nsflag);
-                assert_se(r >= 0);
-                _exit(r);
+                assert_se(r >= 0 || ERRNO_IS_NEG_PRIVILEGE(r));
+                _exit(r >= 0 ? r : EACCES);
```

Comment:
urks, errnos as exit statuses? we so far didn't do that, maybe use the BSD exit status NOPERM for this